### DISCUSSION TITLE
Icons in dark mode

### DISCUSSION THREAD

**Discussion by HectorSancho** [Telefonica/mistica-design#1790] [Proposals]
### Goal

Improve A11y of icons and circle icons in dark mode.

### Context

Some variants are not very accessible.

### Impact

Improving user accessibility in these cases.

### Discussion Questions

For now I think we should only improve the case of new O2 in the circle icon.

### Do you have a visual proposal?

https://www.figma.com/design/qNRch2ojBcGXiMmMFUK0Dy/Asset?node-id=1096-18078&t=0J4KPlZrjxwraxyh-1

**Comment by HectorSancho** [comment]
The first step would be to change the inverse token to grey20 in dark mode.

Brand was so clear because of the avatar component, but with the new tokens I think we could change the brand colour to darkModeBeyondBlue.

![Captura de pantalla 2024-05-31 a las 12 12 39](https://github.com/Telefonica/mistica-design/assets/56122343/bcb1e331-5870-458e-a861-5f98f09dca9b)

![Captura de pantalla 2024-05-31 a las 12 21 37](https://github.com/Telefonica/mistica-design/assets/56122343/6b405689-2849-47ee-b683-454c16505132)

**Comment by aweell** [comment]
Hi @HectorSancho, we've reviewed this and the only remaining failing color is vivo-new, we'll create an issue to fix this:

https://github.com/Telefonica/mistica-design/issues/2608
